source/universal/components/Dashboard/index.js

```diff
@@ -9,15 +9,16 @@ class Dashboard extends Component {
   static contextTypes = {
     logout: PropTypes.func.isRequired,
     syncRepos: PropTypes.func.isRequired,
+    addTag: PropTypes.func.isRequired,
   };
 
   render() {
-    const {logout, syncRepos} = bindActionCreators(this.context, this.props.dispatch);
+    const {logout, syncRepos, addTag} = bindActionCreators(this.context, this.props.dispatch);
```

Comment:
FYI, `bindActionCreators` will automatically wrap `dispatch` around action creator, so after binding, you will not need `dispatch` anymore. Details see http://redux.js.org/docs/api/bindActionCreators.html